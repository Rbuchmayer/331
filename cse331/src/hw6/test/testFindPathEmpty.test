LoadGraph empty emptyGraph.tsv
FindPath empty a a
FindPath empty a b