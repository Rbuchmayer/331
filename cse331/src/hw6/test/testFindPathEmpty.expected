# look for paths in an empty graph
loaded graph empty
unknown character a
unknown character a
unknown character a 
unknown character b